internal/storage/sql/evaluation_test.go

```diff
@@ -682,3 +692,95 @@ func (s *DBTestSuite) TestGetEvaluationRollouts_NonDefaultNamespace() {
 	assert.Equal(t, segment.MatchType, evaluationRollouts[1].Segment.MatchType)
 	assert.True(t, evaluationRollouts[1].Segment.Value, "segment value is true")
 }
+
+func Benchmark_EvaluationV1AndV2(b *testing.B) {
+	s := new(DBTestSuite)
+
+	t := &testing.T{}
+	s.SetT(t)
+	s.SetupSuite()
+
+	server := server.New(zap.NewNop(), s.store)
+	importer := ext.NewImporter(server)
+	reader, err := os.Open("./testdata/benchmark_test.yml")
+	defer func() {
+		_ = reader.Close()
+	}()
+
+	require.NoError(t, err)
+
+	err = importer.Import(context.TODO(), reader)
+	require.NoError(t, err)
+
+	flagKeys := make([]string, 0, 10)
+
+	for i := 0; i < 10; i++ {
+		var flagKey string
+
+		num := rand.Intn(50)
```

Comment:
```suggestion
		num := rand.Intn(50)+1
```

nit: would defeat the need of doing the `if num == 0` check below